modules/classes.js

```diff
@@ -0,0 +1,70 @@
+import BookList from './booklist.js';
+
+export const main = document.getElementById('main');
+export const sectionTitle = document.createElement('h1');
+export const bookListSection = document.createElement('section');
+export const booksList = document.createElement('div');
+
+export const bookList = new BookList();
+
+export class Book {
+  constructor(title, author) {
+    this.id = this.idMaker().next().value;
+    this.title = title;
+    this.author = author;
+  }
+
+  // eslint-disable-next-line
+  * idMaker() {
+    let id;
+    if (localStorage.getItem('bookId')) {
+      id = parseInt(localStorage.getItem('bookId'), 10);
+    } else {
+      id = 0;
+    }
+    while (true) yield (id += 1);
+  }
+
+  addBook() {
```

Comment:
- [ ] As mentioned in the Project requirements we have to use ES6 syntax **"Refactor your methods and functions to arrow functions."** please update all your functions to use arrow functions syntax.

Reply:
done